example/example_gles3.c

```diff
@@ -127,9 +127,11 @@ int main()
 		glEnable(GL_CULL_FACE);
 		glDisable(GL_DEPTH_TEST);
 
+		int pickedID = nvgHitTest(vg, mx, my, NVG_TEST_ALL);
```

Comment:
Move just before nvgEndFrame(). Demos should show case good usage patterns. All per frame stuff so go between nvgBeginFrame()/nvgEndFrame(). Even if the current implementation does not kill anything at end frame, we might later do so.